external/storm-eventhubs/pom.xml

```diff
@@ -21,19 +21,19 @@
     <parent>
         <artifactId>storm</artifactId>
         <groupId>org.apache.storm</groupId>
-        <version>1.0.3-SNAPSHOT</version>
+        <version>1.0.2</version>
```

Comment:
Reverted them. I had initially changed it to validate compatibility with 1.0.2 binaries.